Dashboard/templates/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+
+    <!-- custom css -->
+    <link rel="stylesheet" type="text/css" href="../static/styles/styles.css">
+
+    <!-- Bootstrap css -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <title>Scan8</title>
+</head>
+
+<body>
+    <h1>Scan8</h1>
+    <button type="button" name="button" class="btn btn-outline-secondary">New Scan</button>
+
+    <h4>Queued Scans</h4>
+    <div class="list">
+        <div class="scanItem">
+            <div class="column">
```

Comment:
It would be easy if we use row and col class of bootstrap to make it responsive

Reply:
Sure